src/views/loginView/LoginView.tsx

```diff
@@ -98,7 +98,7 @@ const LoginView: React.FC<any> = props => {
           </p>
           <a
             id='login-feide'
-            href='/login/dataporten_feide/'
+            href='/oidc//login/dataporten_feide/'
```

Comment:
Weird. When I attempt to edit the file via GitHub, the double '//' is simplified to a single '/'.